### ISSUE TITLE
[PM-1221] source tar balls have .git files

### ISSUE THREAD

**Issue by mayani** [pegasus-isi/pegasus#1335]
Some comments from the Debian side (please share them accordingly).

pegasus-source-4.8.0.tar.gz
- contains a full copy of .git/ (making the tarball 2.5 times the necessary size)
- doesn't contain debian/changelog (but debian/changelog.in), and I cannot find
 a rule to create one
- unpacks to pegasus-source-4.8.0 (I'd expect pegasus-4.8.0)

A "stripped" source tarball can be extracted from pegasus-4.8.0.src.rpm
- that doesn't have .git inside
- that comes with a debian/changelog file (build version is -1+deb9, btw)
- that still unpacks to pegasus-source-4.8.0

So here's what I did:

- fetch .src.rpm
- extract .tar.gz (and ignore .spec)
- rename to pegasus_4.8.0.orig.tar.gz
- unpack, and rename directory to pegasus-4.8.0
- edit debian/changelog so the first entry reads "pegasus (4.8.0-1)"
- process the package similar to the LSCsoft ones (to create a deb8u0 and deb9u0 source package
 to be handled be the respective pbuilders)

First attempt to build fails as docbook still is expected to come from the net -
which is "verboten" for pbuilder. Also, a patch has still to be applied to avoid
overwriting of debian/changelog.

**Reporter:** @duncan-brown
**Resolution:** Fixed
**Watchers:**
@rynge
@vahi

**Comment by mayani** [comment]
**Author:** @rynge

I think we have now addressed all the issues:

1) The source package will be named pegasus-N.N.N.tar.gz and it will untar as pegasus-N.N.N

2) The git dirs/files are correctly cleaned up before creating the source package.

3) A debian/changelog is contained in the package, with last entry having a version like 4.8.0-1. The changelog will not be overwritten by ant.

4) The pbuilder network connections is probably solved. I added a few docbook dependencies to the control file, and the local docbook style
sheets should mean docbook would not need to download anything. 

We are awaiting feedback from the user who reported the issue.
